.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "commentTranslate.source": "Bing"

```

Comment:
I'm not sure we need this in Git. I guess `.vscode` should be added to the git ignore file. Everything else seems okay.

Reply:
Yes, just ignore it，